src/FSharpPlus/Data/Matrix.fs

```diff
@@ -0,0 +1,655 @@
+namespace FSharpPlus.Data
+
+open System.Runtime.CompilerServices
+open FSharpPlus.Control
+open FSharpPlus.TypeLevel
+open TypeLevelOperators
+open FSharpPlus.Internals.Prelude
+
+// Items : 'Item[ 'Column, 'Row ]
+[<Struct; StructuredFormatDisplay("{Items}")>]
+type Matrix< 'Item, 'Row, 'Column > = private { Items: 'Item[,] } with
+  [<MethodImpl(MethodImplOptions.AggressiveInlining)>]
+  member this.UnsafeGet (i, j) = this.Items.[i, j]
+  [<MethodImpl(MethodImplOptions.AggressiveInlining)>]
+  static member UnsafeCreate (_row: 'm, _column: 'n, items: _[,]) : Matrix<_, 'm, 'n> =
+    { Items = items }
+  interface System.Collections.Generic.IReadOnlyCollection<'Item> with
```

Comment:
Can we comment out these interfaces implementation for now, until we merge #207 ?

Reply:
OK!

Reply:
Actually we can leave the interface's members, there's no harm at all.
I propose just leave commented out the interfaces declaration, that will suffice for now.

Reply:
Now that I think, since F# has not implicit interfaces, it would create a breaking change, as those methods would become available without casting to the interface.

So better to leave the whole commented out as you already did.